extensions/linear/src/components/IssueActions/CopyToClipboardSection.tsx

```diff
@@ -18,36 +18,36 @@ export default function CopyToClipboardSection({ issue }: { issue: IssueResult }
   return (
     <ActionPanel.Section>
       <Action.CopyToClipboard
-        icon={Icon.Clipboard}
         content={issue.identifier}
         title="Copy Issue ID"
         shortcut={{ modifiers: ["cmd"], key: "." }}
       />
 
       <Action.CopyToClipboard
-        icon={Icon.Clipboard}
-        content={issue.url}
-        title="Copy Issue URL"
-        shortcut={{ modifiers: ["cmd", "shift"], key: "," }}
+        content={{
+          html: `<a href="${issue.url}" title="${issue.title}">${issue.identifier}: ${issue.title}</a>`,
+          text: issue.url,
+        }}
+        title="Copy Formatted Issue URL"
+        shortcut={{ modifiers: ["cmd", "shift"], key: "'" }}
       />
 
+      <Action.CopyToClipboard content={issue.url} title="Copy Issue URL" />
```

Comment:
The new action ("Copy Formatted Issue URL") now uses this shortcut. I figured it's much nicer to copy the formatted one when possible. You think we should put it back?

Reply:
It was `,` and the new one is `'`, isn't it?

Reply:
🤔 Let me double check. Maybe I copy-pasted the wrong one. Totally possible.

Reply:
BTW we could also copy `HTML` and `text` at the same time with the existing action. This way the receiving app decides what it can handle best. Slack would take the HTML but something like VS Code might fallback to the text.

```suggestion
      <Action.CopyToClipboard content={{ text: issue.url, html: `<a href="${issue.url}">{${issue.identifier}: ${issue.title}</a>`} title="Copy Issue URL" />
```

Reply:
@peduarte did you consider my suggestion? Not sure if you have seen it because it's in a resolved conversation.